modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java

```diff
@@ -131,14 +137,29 @@ public static void setUpClass() throws Exception {
 		).put(
 			"${portalURL}",
 			() -> {
-				Company company = _companyLocalService.getCompany(
-					TestPropsValues.getCompanyId());
+				HttpServletRequest httpServletRequest =
+					_originalHttpServletRequest =
```

Comment:
You are assignment a variable and initializing the variable in the same line. This is possible but I've never seen this in Liferay. Could you show me an example, please?
I would prefer to split this code into multiple lines so you can assign the variable and initialize it afterward. What do you think?

Reply:
Okay, I did that!